assets/terraform-modules/controller/variables.tf

```diff
@@ -0,0 +1,85 @@
+variable "dns_zone" {
+  type = string
+}
+
+variable "bootkube_image_name" {
+  description = "Docker image name to use for container running bootkube"
```

Comment:
Like others can you please add period(.) at the end of the description here and other occurrences, in the same file